vhdbuilder/packer/install-dependencies.sh

```diff
@@ -459,15 +459,11 @@ done
 # v1.17.16
 # v1.18.10
 # v1.18.14
-# v1.19.3
 # v1.19.6
-# v1.20.0
+# v1.19.7
+# v1.20.2
 # NOTE that we only keep the latest one per k8s patch version as kubelet/kubectl is decided by VHD version
 K8S_VERSIONS="
-1.16.9-hotfix.20200529.1
```

Comment:
We are removing k8s 1.16 because it is planned to be deprecated in Jan 2021 per [AKS Kubernetes Release Calendar](https://docs.microsoft.com/en-us/azure/aks/supported-kubernetes-versions#aks-kubernetes-release-calendar), or anywhere to track this?

Reply:
Yes 1.16 will be removed from supported versions in this week's release

Reply:
can we keep 1.16.10 for a while if disk space allowed?

keep in mind there is always vmas case where people run unsupported version and relies on disk for optimization.

Reply:
keeping 1.16 for now. removed a few other duplicates